app/Http/Controllers/GeneralLedgerAccountsController.php

```diff
@@ -34,52 +36,87 @@ public function create()
      * @return \Illuminate\Http\Response
      */
     public function store(Request $request)
-    {
-        //
-    }
+    {}
```

Comment:
as though i saw a add new account link on the screen shot? but this blank

Reply:
custom url. need parent account data.